adapters/repos/db/lsmkv/memtable_flush.go

```diff
@@ -108,6 +118,17 @@ func (m *Memtable) flush() error {
 		return err
 	}
 
+	err = os.Rename(tmpSegmentPath, strings.TrimSuffix(tmpSegmentPath, ".tmp"))
+	if err != nil {
+		return err
+	}
+
+	// fsync parent directory
```

Comment:
I don't think we need this `fsync`? no? going to add more latency to the deactivation flow only I think.

Reply:
yes, fsync on the parent directory is needed after renaming the file